### PR TITLE
Bookmark page

### PR DESCRIPTION
**Pull request by Ju-Schn**
Added Page for pinned cards, react router, navbar and updated styling. Also updated specs and conditional rendering of pin marker.

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -3,26 +3,34 @@ import ScreenReaderOnly from './ScreenReaderOnly';
...
+            height="32px"
+            fill={isPinned ? '#415E02' : '#8c0e03'}
+          >
+            <g id="surface14398506">
+              <path d="M 18.9375 0.21875 C 18.582031 0.148438 18.113281 0.382812 17.78125 0.8125 L 13.09375 7.40625 C 10.394531 6.640625 7.347656 7.308594 5.21875 9.4375 L 16.5625 20.75 C 18.71875 18.597656 19.378906 15.539062 18.5625 12.8125 L 18.59375 12.78125 L 25.1875 8.28125 C 25.769531 7.847656 25.980469 7.199219 25.625 6.84375 L 19.25 0.40625 C 19.164062 0.316406 19.054688 0.242188 18.9375 0.21875 Z M 18.9375 3.21875 L 20.6875 4.9375 L 16.21875 9 L 15.1875 8.03125 Z M 13.03125 9.28125 C 13.195312 9.28125 13.28125 9.3125 13.28125 9.3125 L 14.28125 10.28125 C 14.28125 10.28125 13.601562 10.371094 11.5625 12.40625 L 9.78125 10.625 C 11.34375 9.460938 12.539062 9.285156 13.03125 9.28125 Z M 8.625 14.78125 L 0.21875 25.78125 L 11.1875 17.34375 Z M 8.625 14.78125 " />
+            </g>
+          </svg>
+          <ScreenReaderOnly>Pinne diese Karte</ScreenReaderOnly>
+        </StyledButton>
+      )}{' '}
```
Comment (ttaannjjaa) [inline src/components/Card.js:33]: probably the linter put it in there and you don't need it

File: src/index.js [hunk 2]
```diff
@@ -3,11 +3,14 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import GlobalStyles from './GlobalStyles.js';
+import { BrowserRouter } from 'react-router-dom';
 
 ReactDOM.render(
   <React.StrictMode>
     <GlobalStyles />
-    <App />
+    <BrowserRouter>
```
Comment (ttaannjjaa) [inline src/index.js:11]: for my opinion you should put the GlobalStyles inside the BrowserRouter

File: src/pages/CreateCard.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  return (
+    <>
+      <StyledForm
+        onSubmit={handleSubmit}
+        aria-labelledby="create-card"
+        autoComplete="off"
+        name="create"
+      >
+        <h2 id="create-card">
+          {' '}
```
Comment (ttaannjjaa) [inline src/pages/CreateCard.js:16]: Do you need this?

File: src/pages/CreateCard.spec.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import CardForm from './CardForm';
+import CreateCard from './CreateCard';
 
 describe('CardForm', () => {
```
Comment (ttaannjjaa) [inline src/pages/CreateCard.spec.js:5]: I think, you mean here also CreateCard instead of CardForm

File: src/pages/Pinned.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                  onTrashClick={onTrashClick}
+                  onPinClick={onPinClick}
+                  isPinned={card.isPinned}
+                />
+              </li>
+            );
+          else return [];
+        })}
+
+        {showModal && (
```
Comment (ttaannjjaa) [inline src/pages/Pinned.js:34]: when there is a modal, the content outside should not be reachable for the user, important also for accessibility - just as a suggestion for a further userstory

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Really great work - looks very good to me, just a few comments

File: src/components/Card.js (hunk 1 above)
Comment (Ju-Schn) [reply src/components/Card.js:33]: > probably the linter put it in there and you don't need it

Yes, that was the linter, thanks :)

**Pull request merged**